src/utilities/helpers.ts

```diff
@@ -1,41 +1,7 @@
-import { DISABLED, PAGE_REQUEST_SIZE_DEFAULT } from '@/constants'
-import { PaginatedApiListResponse } from '@/types/api.d'
+import { DISABLED } from '@/constants'
 import { ZoneOverview } from '@/types/index.d'
 import { get } from '@/utilities/get'
 
-export async function fetchAllResources<T = Object>(endpoint: (params: Object) => Promise<PaginatedApiListResponse<T>>): Promise<{ items: T[]; total: number }> {
-  try {
-    let allTotal = null
-    let offset = 0
-    let allItems: T[] = []
-    let previousNext: string | null = null
-
-    while (true) {
-      const params = { size: PAGE_REQUEST_SIZE_DEFAULT, offset }
```

Comment:
I think this constant from can be removed the constants file also, up to you if you wanted to do it here or not. I reckon that entire file will be deleted soon anyway. One sec I think I can approve this now with the changes you've made.